pages/stores/[storeId].tsx

```diff
@@ -16,6 +16,7 @@ import {faGlobe} from '@fortawesome/free-solid-svg-icons';
 import {faEnvelope} from '@fortawesome/free-regular-svg-icons';
 import {Link} from '@material-ui/core';
 import useStyles from '../../assets/styles/stores_styles';
+import {analyticsSocialMediaRef} from '../../services/analytics';
```

Comment:
minor: this is formatted differently than in the other files (missing spaces around the import name). You can use the popular [prettier](https://prettier.io/) code formatter to auto-format your code uniformly.

Reply:
Resolved in 324a621